backend/src/personnel/personnel.service.ts

```diff
@@ -192,67 +186,181 @@ export class PersonnelService {
       });
     }
 
+    const { personnel, count} = await this.getPersonnelForProgram<EmcrPersonnelEntity>(
+      'EMCR',
+      qb,
+      query.rows,
+      query.page,
+      query.status
+    );
+
+    return { personnel, count };
+  }
+
+  /**
+   * Get BCWS Personnel
+   * Given specific queries, get associated personnel and their function experiences
+   * @param query Includes pagination query, ie. page and number of rows
+   * @returns {BcwsPersonnelEntity[]} List of personnel
+   * @returns {number} Count of total personnel search applies to
+   */
+  async getBcwsPersonnel(query: GetBcwsPersonnelDTO): Promise<{
+    personnel: BcwsPersonnelEntity[];
+    count: {
+      [Status.ACTIVE]: number;
+      [Status.INACTIVE]: number;
+      [Status.PENDING]: number;
+    }
+  }> {
+    this.logger.log(`Query: ${JSON.stringify(query)}`);
+    const qb =
+      this.bcwsPersonnelRepository.createQueryBuilder('bcws_personnel');
+    qb.leftJoinAndSelect('bcws_personnel.personnel', 'personnel');
+    qb.leftJoinAndSelect('bcws_personnel.roles', 'roles');
+    qb.leftJoinAndSelect('roles.role', 'role');
+    qb.leftJoinAndSelect('bcws_personnel.homeFireCentre', 'homeFireCentre');
+
+    this.addQueryBuilderCommonFilters(qb, query.name, query.availabilityType, query.availabilityFromDate, query.availabilityToDate);
+
+    if (query.fireCentre?.length) {
+      qb.andWhere('bcws_personnel.homeFireCentre.fireCentre IN (:...fireCentres)', {
+        fireCentres: query.fireCentre,
+      });
+    }
+    if (query.location?.length) {
+      qb.andWhere(
+        'bcws_personnel.homeFireCentre.locationName IN (:...homeLocations)',
+        {
+          homeLocations: query.location,
+        },
+      );
+    }
+
+    if (query.section && query.role) {
```

Comment:
I think we can search by role only, as the section is implied by the role, but also totally good to leave it - I wasn't sure if we wanted to pass section along in the query params, or just role.